cfme/utils/ssh.py

```diff
@@ -49,6 +51,10 @@ class SSHResult(namedtuple("SSHResult", ["rc", "output"])):
     But it still subclasses the original class therefore all old behaviour is kept. But you don't
     have to expand the tuple or pull the value out if you are checking only one of them.
     """
+    rc = attr.ib()
```

Comment:
**required** please rename the attributes to `return_code` and `command`

additionally set `repr=False` on `output` and reorder to `command, return_code, output`

that way we get a nicer repr that's readable on terminal

Reply:
@RonnyPfannschmidt Unless you feel strongly about it, I think I'd rather leave `rc` as-is, and not change everything to `return_code` as I think `rc` is descriptive enough in the context to correctly expect/understand what it is.